dev-docs/modules/paapi.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: page_v2
+page_type: module
+title: Module - PAAPI
+description: Protected audience API
+module_code : paapi
+display_name : PAAPI
+enable_download : true
+sidebarType : 1
+---
+
+# PAAPI module
+
+This module allows Prebid.js to support PAAPI, formerly known as [FLEDGE](https://github.com/WICG/turtledove/blob/main/FLEDGE.md).
+
+This document covers the steps necessary for publishers to enable PAAPI on their inventory. It also describes
+the changes Bid Adapters need to implement in order to support PAAPI.
+
+A related module, [fledgeForGpt](/dev-docs/modules/fledgeForGpt.html), adds support specifically for GPT's [component auctions](https://developers.google.com/publisher-tag/reference#googletag.config.componentauctionconfig).
+
+## Publisher Integration
+
+To use PAAPI, publishers must:
+
+- include this module in their Prebid.js bundle
+
+   ```bash
+   gulp build --modules=paapi,...
+   ```
+
+- enable PAAPI, globally or by ad unit, through [configuration](#config)
+- manage the PAAPI auctions. This can be delegated to GPT with the [fledgeForGpt module](/dev-docs/modules/fledgeForGpt.html); homegrown solutions are possible with [getPAAPIConfig](/dev-docs/publisher-api-reference/getPAAPIConfig.html), but out of scope for this document.
+
+<a id="config"></a>
+
+### Module Configuration
+
+This module exposes the following settings:
+
+{: .table .table-bordered .table-striped }
+|Name |Type |Description |Notes |
+| ------------ | ------------ | ------------ |------------ |
+|enabled | Boolean |Enable/disable the module |Defaults to `false` |
+|bidders | Array[String] |Optional list of bidders |Defaults to all bidders |
+|defaultForSlots | Number |Default value for `imp.ext.ae` in requests for specified bidders |Should be 1 |
+
+As noted above, PAAPI support is disabled by default. To enable it, set the `enabled` value to `true` for this module and configure `defaultForSlots` to be `1` (meaning _Client-side auction_).
+using the `setConfig` method of Prebid.js:
+
+```js
+pbjs.que.push(function() {
+  pbjs.setConfig({
+    paapi: {
+      enabled: true,
+      defaultForSlots: 1
+    }
+  });
+});
+```
+
+Optionally, PAAPI support can be limited to specific bidders:
+
+```js
+pbjs.que.push(function() {
+  pbjs.setConfig({
+    paapi: {
+      enabled: true,
+      defaultForSlots: 1,
+      bidders: ['bidderA', 'bidderB']
+    }
+  });
+});
+```
+
+### Bidder Configuration
+
+This module adds the following setting for bidders:
+
+{: .table .table-bordered .table-striped }
+|Name |Type |Description |Notes |
+| ------------ | ------------ | ------------ |------------ |
+| fledgeEnabled | Boolean | Enable/disable a bidder to participate in FLEDGE | Defaults to `false` |
```

Comment:
I assume this can't be renamed since it would break existing deployed configs.
Could a future looking replacement be introduced, e.g. `paapiEnabled`, with `fledgeEnabled` deprecated down the road?